v13.0.11 has apparently broken imports

<!--
Thanks for your interest in the project. I appreciate bugs filed and PRs submitted!
Please make sure that you are familiar with and follow the Code of Conduct for
this project (found in the CODE_OF_CONDUCT.md file).

Please fill out this template with all the relevant information so we can
understand what's going on and fix the issue.

I'll probably ask you to submit the fix (after giving some direction). If you've
never done that before, that's great! Check this free short video tutorial to
learn how: http://kcd.im/pull-request
-->

- `@testing-library/user-event` version: 13.0.11
<!--
If your version doesn't match this project's current release found at
https://github.com/testing-library/user-event/releases, please try upgrading
to see if that solves your issue. This especially may be the case if you are
using Create React App, which currently provides an older version of user-event.
-->

<!--
Keep in mind that if you're using a version of node we don't support that
could also be an issue. Check our package.json "engines" file for the
supported version.
-->

Relevant code or config

https://github.com/ljosberinn/personal-react-boilerplate/pull/905/checks?check_run_id=2187475320

What you did: just a dependabot auto-update

What happened:
Multiple tests fail, all with the same error:

```sh
FAIL src/client/__tests__/useAuth.test.tsx
  ● Test suite failed to run

    Cannot find module 'utils/misc/isElementType' from 'node_modules/@testing-library/user-event/dist/utils/edit/calculateNewValue.js'

    Require stack:
      node_modules/@testing-library/user-event/dist/utils/edit/calculateNewValue.js
      node_modules/@testing-library/user-event/dist/utils/index.js
      node_modules/@testing-library/user-event/dist/click.js
      node_modules/@testing-library/user-event/dist/index.js
      testUtils/index.tsx
      src/client/__tests__/useAuth.test.tsx

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (node_modules/@testing-library/user-event/dist/utils/edit/calculateNewValue.js:8:22)
```

sorry for not providing more info, can't debug at this time

--- a/src/utils/edit/calculateNewValue.ts
+++ b/src/utils/edit/calculateNewValue.ts
@@ -1,4 +1,4 @@
-import {isElementType} from 'utils/misc/isElementType'
+import {isElementType} from '../misc/isElementType'
 import {getSelectionRange} from './selectionRange'
 import {getValue} from './getValue'
 import {isValidDateValue} from './isValidDateValue'

--- a/src/utils/edit/hasUnreliableEmptyValue.ts
+++ b/src/utils/edit/hasUnreliableEmptyValue.ts
@@ -1,4 +1,4 @@
-import {isElementType} from 'utils/misc/isElementType'
+import {isElementType} from '../misc/isElementType'
 
 enum unreliableValueInputTypes {
   'number' = 'number',

--- a/src/utils/edit/selectionRange.ts
+++ b/src/utils/edit/selectionRange.ts
@@ -1,4 +1,4 @@
-import {isElementType} from 'utils/misc/isElementType'
+import {isElementType} from '../misc/isElementType'
 
 // https://github.com/jsdom/jsdom/blob/c2fb8ff94917a4d45e2398543f5dd2a8fed0bdab/lib/jsdom/living/nodes/HTMLInputElement-impl.js#L45
 enum selectionSupportType {

